File: Fundamentos da Programação de Computadores/3. Estrutura sequencial/Exercícios Resolvidos/Ex7.c
//Faça um programa que receba o salário base de um funcionário, calcule e mostre seu salário a receber,
//sabendo-se que o funcionário tem gratificação de R$ 50 e paga imposto de 10% sobre o salário base.
#include <stdio.h>
#include <stdlib.h>

int main()
{
    float salario,salarioReceber = 0;

    scanf("%f",&salario);

    salarioReceber = (salario + 50) - (salario * 0.10f);

    printf("%.2f",salarioReceber);

    return 0;

}// end main

File: Java/Livros/Fundamentos da programação de computadores/Capítulo 3 Estrutura sequencial/Ex7.java
/*Faça um programa que receba o salário base de um funcionário, calcule e mostre seu salário a receber,
sabendo-se que o funcionário tem gratificação de R$ 50 e paga imposto de 10% sobre o salário base.*/

import java.util.Scanner;

public class Ex7 {

    static public void main(String[] args) {
        float salario;
        float gratificacao = 50;

        Scanner dado = new Scanner(System.in);

        System.out.println("Digite Salario: ");
        salario = dado.nextFloat();

        salario = salario + gratificacao - imposto(salario);

        System.out.println("Salario a receber: " + salario);

        dado.close();

    }

    static public float imposto(float salario) {
        return (salario * 0.10f);
    }

}
